doc/users/whats_new/rcparams.rst

```diff
@@ -1,25 +1,27 @@
 Configuration (rcParams)
 ------------------------
 
-+----------------------------+--------------------------------------------------+
-| Parameter                  | Description                                      |
-+============================+==================================================+
-|`date.autoformatter.year`   | foramt string for 'year' scale dates             |
-+----------------------------+--------------------------------------------------+
-|`date.autoformatter.month`  | format string for 'month' scale dates            |
-+----------------------------+--------------------------------------------------+
-|`date.autoformatter.day`    | format string for 'day' scale dates              |
-+----------------------------+--------------------------------------------------+
-|`date.autoformatter.hour`   | format string for 'hour' scale times             |
-+----------------------------+--------------------------------------------------+
-|`date.autoformatter.minute` | format string for 'minute' scale times           |
-+----------------------------+--------------------------------------------------+
-|`date.autoformatter.second` | format string for 'second' scale times           |
-+----------------------------+--------------------------------------------------+
-|`scatter.marker`            | default marker for scatter plot                  |
-+----------------------------+--------------------------------------------------+
-|`svg.hashsalt`              | see note                                         |
-+----------------------------+--------------------------------------------------+
++------------------------------+--------------------------------------------------+
+| Parameter                    | Description                                      |
++==============================+==================================================+
+|`date.autoformatter.year`     | foramt string for 'year' scale dates             |
++------------------------------+--------------------------------------------------+
+|`date.autoformatter.month`    | format string for 'month' scale dates            |
++------------------------------+--------------------------------------------------+
+|`date.autoformatter.day`      | format string for 'day' scale dates              |
++------------------------------+--------------------------------------------------+
+|`date.autoformatter.hour`     | format string for 'hour' scale times             |
++------------------------------+--------------------------------------------------+
+|`date.autoformatter.minute`   | format string for 'minute' scale times           |
++------------------------------+--------------------------------------------------+
+|`date.autoformatter.second`   | format string for 'second' scale times           |
++------------------------------+--------------------------------------------------+
+|`date.autoformatter.musecond` | format string for 'micsecond' scale times        |
```

Comment:
-> 'microsecond' scale
Maybe it is also worth two extra characters to spell out `date.autoformatter.microsecond`.  "musecond" looks odd and will be harder to remember.